.github/workflows/python-tests.yml

```
@@ -0,0 +1,39 @@
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+        matrix:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)